source/admin_manual/clients.rst

```diff
@@ -0,0 +1,233 @@
+===============================
+Client issues and configuration
+===============================
+
+It seems to be quite difficult to implement a working IMAP client. `Best
+Practices for Implementing an IMAP
+Client <http://www.imapwiki.org/ClientImplementation>`__ tries to help
+you with it.
+
+Appsuite webmail
+----------------
+
+Works fine.
+
+
+Apple Mail.app
+--------------
+
+On Mac OS X Leopard 10.5 Mail.app appears to support
+subscribe/unsubscribe by right clicking on a mailbox, selecting 'Get
+Account Info' and selecting 'Subscription List' from tabs. This however
+doesn't really work with any IMAP server.
+
+Apple Mail 3.6 (that comes with OS X 10.5 Leopard) supports
+subscribing/unsubscribing to folders in the public namespace.
```

Comment:
I don't know the details regarding Apple's upgrade policy, but are these still in circulation/supported (both the OS as well as the clients)?

Reply:
what would be a reasonable version number?

Reply:
I just checked and OS X 10.5 is from 2007. If I had to decide I would only keep the list updated for officially supported OS and Mail combinations. On the other hand: I have no Apple device and am not too keen on getting into their update-cycle.

Maybe Timo knows?

On the other other hand: What's the danger in keeping this information on hand. I'll just close this conversation.